src/main/java/org/jenkinsci/plugins/workinghours/ValidationResult.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.workinghours;
+
+import net.sf.json.JSONObject;
+
+public class ValidationResult {
+    public boolean isValid;
+    public String errorMessage;
+    public String fieldName;
```

Comment:
Usually in Java, fields are private or protected, and accessed through public getters. Unless there's a reason that won't work, we should follow that pattern here.